cmd/kpod/rm.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	rmFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Force removal of a running container.  The default is false",
+		},
+		cli.BoolFlag{
```

Comment:
Doesn't look like this flag is implemented. Should either remove it or make it throw an error on being set.

Reply:
the force flag has been implemented. It is used to force remove running and created containers.